packages/babel-helper-create-regexp-features-plugin/src/features.ts

```diff
@@ -3,6 +3,7 @@ export const FEATURES = Object.freeze({
   dotAllFlag: 1 << 1,
   unicodePropertyEscape: 1 << 2,
   namedCaptureGroups: 1 << 3,
+  // Babel 8.1: remove unicodeSetsFlag_syntax
```

Comment:
Why not 8.0?

Reply:
The type `FeatureType` depends on the value here, I was worrying plugging in `BABEL_8_BREAKING` switch here will confuse TypeScript. It seems that it can handle that well.

However, now I think we should preserve this bit even if it is no long used. The bits here decides how we interpret the `filePass.get("@babel/plugin-regexp-features/featuresKey")`. If we preserve this bit, then the featureKey defined by a Babel 7 plugin will still be respected by a Babel 8 regex plugin. Doing so should help the migration as mis-interpreting feature bits will result to confusing regex feature compilation: Babel is transpiling regex features other than what is perceived by users.

Therefore, unless we run out of all 53 bits, I don't think we have to remove any unused syntax bits. It suffices to just note that this bit is not used and is for compatibility with archived plugins.